### PR TITLE
Created a test for checking if calculated balance is correct.

### PR DESCRIPTION
**Pull request by Zoltekk**
This code implements a test which does the following things :
    1. Log into website.
    2. Add 3 random items into the cart.
    3. Proceeds to checkout
    4. Check the sum of the value of the added items against the sum calculated by the website.

### CODE REVIEW

File: task1/config.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "settings": {
+    "baseurl": "https://www.saucedemo.com/"
```
Comment (Glovacky) [inline task1/config.json]: Change it to kebab case please

File: task1/conftest.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import pytest
+import json
+from selenium import webdriver
+
+@pytest.fixture()
+def get_driver():
+    driver = webdriver.Firefox()
+    driver.implicitly_wait(5)
```
Comment (Glovacky) [inline task1/conftest.py]: How about we keep the duration in config?

File: task1/test_task1.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import random
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+import time
+import pytest
+from pprint import pprint
+
```
Comment (Glovacky) [inline task1/test_task1.py:2]: Double check if all the imports are necessary

**Review by Glovacky — CHANGES_REQUESTED** [review]

File: task1/config.json (hunk 1 above)
Comment (Zoltekk) [reply task1/config.json]: Changed

File: task1/conftest.py (hunk 2 above)
Comment (Zoltekk) [reply task1/conftest.py]: Moved to config

File: task1/test_task1.py (hunk 3 above)
Comment (Zoltekk) [reply task1/test_task1.py:2]: Removed unnecessary imports

**Review by Glovacky — APPROVED** [review]
good enough bucko

**Pull request merged**